src/preact.d.ts

```diff
@@ -154,18 +154,7 @@ declare namespace preact {
 		vnode?: (vnode: VNode<any>) => void;
 		event?: (event: Event) => Event;
 	};
-}
-
-type Defaultize<Props, Defaults> =
-	// Distribute over unions
-	Props extends any
-		? 	// Make any properties included in Default optional
-			& Partial<Pick<Props, Extract<keyof Props, keyof Defaults>>>
-			// Include the remaining properties from Props
-			& Pick<Props, Exclude<keyof Props, keyof Defaults>>
-		: never;
 
-declare global {
```

Comment:
removing it would probably be a breaking change, I guess we'll need both and the only way to do it might be to copy paste